arrow/src/main/java/org/apache/calcite/adapter/arrow/ArrowTranslator.java

```diff
@@ -228,6 +228,8 @@ private static String getLiteralType(Object literal) {
       }
     } else if (String.class.equals(literal.getClass())) {
       return "string";
+    } else if (literal instanceof Double) {
+      return "float";
```

Comment:
why `float`?

Reply:
this is the same string that is used 3 lines above